### PR TITLE
Character consolidation

### PR DESCRIPTION
**Pull request by JFranklin3000**
Closes [issue URL]

##### In this PR
- removed GI Joe, Power Ranger, Pony, and Transformer Actor types
- added a Player Character Actor Type
- migration for existing characters requires ``game.settings.set("essence20", "systemMigrationVersion", "4.4.0")``

##### Testing
- create a character of each type before pulling this down. 
- run  ``game.settings.set("essence20", "systemMigrationVersion", "4.4.0")``
- refresh and validate that all of them are now PlayerCharacters.

### CODE REVIEW

File: templates/dialog/crossover-options.hbs [hunk 1]
```diff
@@ -1,32 +1,32 @@
 <form autocomplete="off" onsubmit="event.preventDefault();">
   <div class="flexcol">
-    {{#inArray '["powerRanger"]' actor.type}}
+    {{#if system.CanMorph}}
     {{else}}
```
Comment (phildominguez) [inline templates/dialog/crossover-options.hbs:4]:
```suggestion
    {{#unless system.CanMorph}}
```

File: templates/dialog/crossover-options.hbs [hunk 2]
```diff
@@ -1,32 +1,32 @@
...
         <input type="checkbox" id="canMorph" name="system.canMorph" {{checked system.canMorph}} />
       {{/inline}}
       {{/"systems/essence20/templates/item/parts/sheet-field.hbs"}}
-    {{/inArray}}
+    {{/if}}
   </div>
   <div class="flexcol">
-    {{#inArray '["pony", "npc"]' actor.type}}
+    {{#if system.canSpellcast}}
     {{else}}
```
Comment (phildominguez) [inline templates/dialog/crossover-options.hbs:14]:
```suggestion
    {{#unless system.canSpellcast}}
```

File: templates/dialog/crossover-options.hbs [hunk 3]
```diff
@@ -1,32 +1,32 @@
...
         <input type="checkbox" id="canSpellcast" name="system.canSpellcast" {{checked system.canSpellcast}}/>
       {{/inline}}
       {{/"systems/essence20/templates/item/parts/sheet-field.hbs"}}
-    {{/inArray}}
+    {{/if}}
   </div>
   <div class="flexcol">
-    {{#ifEquals actor.type 'transformer'}}
+    {{#if system.canTransform}}
     {{else}}
```
Comment (phildominguez) [inline templates/dialog/crossover-options.hbs:24]:
```suggestion
    {{#unless system.canTransform}}
```

File: module/sheet-handlers/role-handler.mjs [hunk 4]
```diff
@@ -328,6 +328,16 @@ export async function onRoleDrop(actor, role, dropFunc) {
     });
   }
 
+  if (role.system.version =='powerRangers') {
```
Comment (phildominguez) [inline module/sheet-handlers/role-handler.mjs:331]: Forgot to handle TF

**Review by phildominguez — CHANGES_REQUESTED** [review]

File: templates/dialog/crossover-options.hbs (hunk 1 above)
Comment (JFranklin3000) [reply templates/dialog/crossover-options.hbs:4]: closing as this will be rewritten

File: templates/dialog/crossover-options.hbs (hunk 2 above)
Comment (JFranklin3000) [reply templates/dialog/crossover-options.hbs:14]: closing as this will be rewritten

File: templates/dialog/crossover-options.hbs (hunk 3 above)
Comment (JFranklin3000) [reply templates/dialog/crossover-options.hbs:24]: closing as this will be rewritten

**Review by phildominguez — APPROVED** [review]

**Pull request merged**